ui/src/components/Configuration/ConfigurationToast.vue

```diff
@@ -0,0 +1,136 @@
+<template>
```

Comment:
This toast was built prior to the latest release of feather. The newer releases contain the new snackbar (notification) component.
This toast file, along with it's hook and anything else related to it, can be removed in favour of the one in dev. The one in dev uses a simple `useSnackbar` hook to open the notification.

Reply:
'ProvisionDConfig.vue' uses it. I'm not certain about its removal. Do you mean to replace it `snackbar`?

Reply:
Yup, remove the toast and wherever a msg needs to be displayed, replace it with the snackbar function instead